app/controllers/users/sessions_controller.rb

```diff
@@ -31,10 +31,9 @@ def active
     end
 
     def timeout
-      if sign_out
-        analytics.track_event(Analytics::SESSION_TIMED_OUT)
-        flash[:timeout] = t('session_timedout')
-      end
+      analytics.track_event(Analytics::SESSION_TIMED_OUT)
+      sign_out
```

Comment:
Do you mean if someone manually types `/timeout` in their URL bar? Is there a scenario where the app calls the `/timeout` endpoint when the user is already signed out?